libqtile/core/manager.py

```diff
@@ -179,6 +181,15 @@ def load_config(self, initial: bool = False) -> None:
         if "resume" in hook.subscriptions:
             subscribe_for_resume_events()
 
+        # To ensure compliance with X11's window layering spec
+        # We need to restack fullscreen windows if they lose focus
+        if self.core.name == "x11":
+            hook.subscribe.client_focus(self.check_stacking)
+
```

Comment:
In line 167 just above we inform the core that the config is being loaded, so could we move this block into `Core.on_config_loaded` in the X backend?

Similarly, as `self.check_stacking` is doing something for the benefit of the X window layer spec, we can move that to the Core too. This hides the X11-specific `change_layer` window method inthe backend, so we can drop that from the base Window class.